tools/small_rna_maps/small_rna_maps.py

```diff
@@ -54,15 +54,13 @@ def create_map(self, bam_object, minsize, maxsize):
             map_dictionary[(chrom, 1, 'F')] = []
             map_dictionary[(chrom, self.chromosomes[chrom], 'F')] = []
             for read in bam_object.fetch(chrom):
-                if (read.query_alignment_length >= minsize and
-                        read.query_alignment_length <= maxsize):
```

Comment:
The reason a test output was changed is that the previous version of small_rna_maps filtered not on the read length, as said in the documentation, but on the alignment length.